internal/capture/capture.go

```diff
@@ -317,6 +334,12 @@ func k8sIPs(addr string) []string {
 
 	namespace, selectorType, selectorValue := sections[0], sections[1], sections[2]
 
+	sectionsSize := len(sections)
+
+	if sectionsSize > 3 {
+		selectorValue = strings.Join(sections[sectionsSize-2:], "/")
```

Comment:
```suggestion
		selectorValue = strings.Join(sections[2:], "/")
```

Reply:
Minor suggestion, this way it should also work if the selectorValue has more than 1 slash